pennylane/ops/qubit.py

```diff
@@ -1882,6 +1882,39 @@ def diagonalizing_gates(self):
         return [QubitUnitary(self.eigendecomposition["eigvec"].conj().T, wires=list(self.wires))]
 
 
+# =============================================================================
```

Comment:
Worth adding a new section below https://pennylane.readthedocs.io/en/latest/introduction/operations.html#qubit-observables for measurements?

E.g. "Qubit Measurements"

Reply:
For sure!